lib/customer.rb

```diff
@@ -0,0 +1,59 @@
+require_relative '../lib/order'
+require 'csv'
+require 'awesome_print'
+module Grocery
+
+  class Customer < Grocery::Order
+      attr_reader :id, :email, :address1, :city, :state, :zipcode
```

Comment:
I don't think that `Customer` should inherit from `Order`. They don't share much functionality.

A good test of whether inheritance is the right tool is saying the sentence "a `Customer` _is an_ `Object`". If that doesn't seem right, then inheritance isn't the right choice.